### DISCUSSION TITLE
lazy column / virtual scroller solution

### DISCUSSION THREAD

**Discussion by tkkcc** [DioxusLabs/dioxus#1000] [Q&A]
In order to render very long list efficiently. In jetpack compose, we have LazyColumn. In react/vue, we have virtual scroller. There seems no solution in dioxus currently.

**Comment by ealmloff** [comment]
A renderer agnostic virtualized list is blocked on some improvements to #894. We need to a expose cross platform equivalent to the intersection observer web API
